### PR TITLE
Update vo.md

### PR DESCRIPTION
**Pull request by zhengj2007**
Updated license and homepage link and added usage

### CODE REVIEW

File: ontology/vo.md [hunk 1]
```diff
@@ -25,6 +25,29 @@ publications:
   title: Mining of vaccine-associated IFN-γ gene interaction networks using the Vaccine Ontology
 repository: https://github.com/vaccineontology/VO
 tracker: https://github.com/vaccineontology/VO/issues
+usages:
```
Comment (cthoyt) [inline ontology/vo.md:28]: I'm not the right one to comment on if the usages are written correctly or not

**Review by yongqunh — APPROVED** [review]
I approve the udpates.

File: ontology/vo.md [hunk 2]
```diff
@@ -25,6 +25,29 @@ publications:
...
+- description: VIOLIN uses VO to standardize vaccine information
+  examples:
+  - description: VIOLIN using VO grouped all SARS-CoV-2 vaccines
+    url: https://violinet.org/canvaxkb/vaccine_detail.php?c_vaccine_id=5339
+  - description: A specific vaccine ‘Allogeneic Tumor Cell Vaccine’ curated in VO for VIOLIN vaccine record
+    url: https://violinet.org/vaxquery/query_detail.php?c_pathogen_id=321#vaccine_5878
```
Comment (deepakunni3) [inline ontology/vo.md:34]: There doesn't seem to be anything rendered as results for this query URL.
Is there some issues on the backend that is affecting the search and retrieval?

File: ontology/vo.md [hunk 3]
```diff
@@ -25,6 +25,29 @@ publications:
...
+  user: https://www.niaid.nih.gov/research/vaccine-adjuvant-compendium-vac
+- description: ImmPort uses Vaccine Ontology to standardize vaccine recorded collected in NIH funded ImmPort studies
+  examples:
+  - description: Vaccine annotated using VO terms
+    url: https://www.immport.org/shared/home
+  user: https://www.immport.org/
+- description: Human Immunology Project Consortium (HIPC) uses VO to standardize vaccine records
+  examples:
+  - description: standardize vaccine records using VO
+    url: https://immunespace.org/
```
Comment (deepakunni3) [inline ontology/vo.md:49]: This seems to generic of a usage. Would it be possible to have a direct link that can illustrate the use of a VO term?

File: ontology/vo.md [hunk 4]
```diff
@@ -25,6 +25,29 @@ publications:
   title: Mining of vaccine-associated IFN-γ gene interaction networks using the Vaccine Ontology
 repository: https://github.com/vaccineontology/VO
 tracker: https://github.com/vaccineontology/VO/issues
+usages:
+- description: VIOLIN uses VO to standardize vaccine information
+  examples:
+  - description: VIOLIN using VO grouped all SARS-CoV-2 vaccines
+    url: https://violinet.org/canvaxkb/vaccine_detail.php?c_vaccine_id=5339
```
[the remark this answers is not in the document]
Comment (zhengj2007) [reply ontology/vo.md:32]: Due to the system update, the link did not work. The issue has been resolved. The URL is working now.

File: ontology/vo.md (hunk 2 above)
Comment (zhengj2007) [reply ontology/vo.md:34]: Due to the system update, the link did not work. The issue has been resolved. The URL is working now.

File: ontology/vo.md [hunk 5]
```diff
@@ -25,6 +25,29 @@ publications:
...
+  user: https://violinet.org
+- description: Vaccine Adjuvant Compendium (VAC) uses Vaccine Ontology to standard vaccine adjuvants developed by NIH
+  examples:
+  - description: A specific vaccine adjuvant, such as CaPNP (CaPtivant)(TM), in Vaccine Adjuvant Compendium, uses VO_0005295 ‘CaPNP (CaPtivant)(TM) vaccine adjuvant’
+    url: https://vac.niaid.nih.gov/view?id=11
+  user: https://www.niaid.nih.gov/research/vaccine-adjuvant-compendium-vac
+- description: ImmPort uses Vaccine Ontology to standardize vaccine recorded collected in NIH funded ImmPort studies
+  examples:
+  - description: Vaccine annotated using VO terms
+    url: https://www.immport.org/shared/home
```
[the remark this answers is not in the document]
Comment (zhengj2007) [reply ontology/vo.md]: The VO terms are showed only when users download a data file that required having an ImmPort account. So, it is hard to provide a link with VO terms. Added an ImmPort data model that shows VO is used.

File: ontology/vo.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (deepakunni3) [reply ontology/vo.md:32]: Perfect! Thank you for clarifying :)

File: ontology/vo.md (hunk 3 above)
Comment (deepakunni3) [reply ontology/vo.md]: awesome! Thank you

File: ontology/vo.md (hunk 2 above)
Comment (deepakunni3) [reply ontology/vo.md:34]: Ah yes. The link works now. Thank you for clarifying!

**Review by matentzn — APPROVED** [review]
Very nice documentation of verifiable usages.

**Pull request merged**